packages/flutter/lib/src/material/drawer.dart

```diff
@@ -82,18 +82,28 @@ class Drawer extends StatelessWidget {
 /// * [Drawer]
 /// * [Scaffold.drawer]
 class DrawerController extends StatefulWidget {
+  /// Creates a controller for a [Drawer].
```

Comment:
reading the class description and the constructor description I still don't know what a DrawerController is or why I would use one.